### ISSUE TITLE
Request: Option to Always Show C/C++ Configuration in Status Bar

### ISSUE THREAD

**Issue by benjaminc247** [microsoft/vscode-cpptools#5822]
As of the latest release I am able to tie my build task and launch command back to the c/c++ configuration, which is awesome!

I have one problem though, the c/c++ configuration does not show up in the status bar unless I have c/c++ files open. It would be nice to be able to quickly change configurations, build, and run without necessarily opening a file. I realize I can select the configuration from the command palette, but I'm after convenience here.

Would it be possible to add a setting (which I would set in my workspace) to always show the c/c++ configuration in the status bar?

**Comment by sean-mcmanus** [comment]
Yeah, I've hit this issue too, mostly when my cursor is the Output logging.

The code that filters by cpp is at https://github.com/microsoft/vscode-cpptools/blob/release/Extension/src/LanguageServer/ui.ts#L164 .

So it would be pretty trivial to add/remove some filter at that spot (if the file types are general enough to be added without a setting), but more work to add a setting to always show it (if anyone wanted to make PR for the change).
